src/parser/jobs/mnk/modules/Undesirables.js

```diff
@@ -0,0 +1,102 @@
+import React, {Fragment} from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS, {getAction} from 'data/ACTIONS'
+import Module from 'parser/core/Module'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {Table} from 'semantic-ui-react'
+
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+const UNDESIRABLES = [
+	ACTIONS.EARTH_TACKLE.id,
+	ACTIONS.ONE_ILM_PUNCH.id,
+]
+
+export default class Undesirables extends Module {
+	static handle = 'undesirables'
+	static dependencies = [
+		'suggestions',
+	]
+
+	static title = 'Undesirable Skills'
+	static displayOrder = DISPLAY_ORDER.UNDESIRABLES
+
+	_undesirables = []
+
+	constructor(...args) {
+		super(...args)
+
+		this.addHook('cast', {by: 'player', abilityId: UNDESIRABLES}, this._onBad)
+		this.addHook('complete', this._onComplete)
+	}
+
+	_onBad(event) {
+		const actionId = event.ability.guid
+
+		// WTF but always worth checking
+		if (!actionId) {
+			return
+		}
+
+		this._undesirables.push({id: actionId, timestamp: event.timestamp})
+	}
+
+	_onComplete() {
+		const oneIlmPunchCount = this._undesirables.filter(event => event.id === ACTIONS.ONE_ILM_PUNCH.id).length
+		const earthTackleCount = this._undesirables.filter(event => event.id === ACTIONS.EARTH_TACKLE.id).length
+		const lostTacklePotency = earthTackleCount * (ACTIONS.FIRE_TACKLE.potency - ACTIONS.EARTH_TACKLE.potency)
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: ACTIONS.EARTH_TACKLE.icon,
+			content: <Fragment>
```

Comment:
Non-blocking, but `<Fragment>...</Fragment>` can (should?) be expressed as `<>...</>`. There's existing uses of `<Fragment>` in the codebase from before when `<>` was supported, but I'd like to ratchet the metric if possible.